verticapy/core/vdataframe/_corr.py

```diff
@@ -1028,6 +1028,9 @@ def corr_pvalue(
             )
             x = val * val * n * min(k, r)
             pvalue = scipy_st.chi2.sf(x, (k - 1) * (r - 1))
+        else:
+            raise ValueError(f"{method} is not a valid method.")
```

Comment:
There was a pylint error complaining that  pvalue might not have been initialized.